lib/helpers/functions.ts

```diff
@@ -0,0 +1,5 @@
+import {ABBREVIATION_INDEX, INTERLEX_INDEX} from "../settings.ts";
+import {CDEStatus, CDEType} from "../models.ts";
+
+export const getRowMappingStatus = (row: string[]) => row[ABBREVIATION_INDEX] ? CDEStatus.Mapped : CDEStatus.Unmapped;
```

Comment:
I did not include this has part of the datasetMapping in the context because with the new modelling of not using an object but rather an array of strings (where we use the index of the header to access the content) it could be tricky to know how to access this data. One option would be to agree that they would be the last 2 columns but that would add a bit of complexity when we need to update the headers so for now I opted for this.

Reply:
@afonsobspinto imo we should try to keep the CDE dialog "polymorphic"